### ISSUE TITLE
Build failed 5.0.2 due to SwiftLint superfluous rules

### ISSUE THREAD

**Issue by aabanaag** [malcommac/SwiftDate#560]
Having issues with the recently released build.

```
$ carthage update --platform iOS
Please update to the latest Carthage version: 0.30.1. You currently are on 0.29.0
*** Fetching Toucan
*** Fetching Alamofire
*** Fetching QRCode
*** Fetching Disk
*** Fetching RxSwift
*** Fetching SwiftyJSON
*** Fetching SwiftIconFont
*** Fetching PopupDialog
*** Fetching Lumina
*** Fetching SwiftDate
*** Fetching SVProgressHUD
*** Fetching DZNEmptyDataSet
*** Fetching SkeletonView
*** Checking out SkeletonView at "1.2.1"
*** Checking out DZNEmptyDataSet at "v1.8.1"
*** Checking out SwiftDate at "5.0.2"
*** Checking out SVProgressHUD at "2.2.5"
*** Checking out Lumina at "1.3.1"
*** Checking out PopupDialog at "0.5.4"
*** Checking out SwiftIconFont at "2.8.0"
*** Checking out SwiftyJSON at "4.1.0"
*** Checking out RxSwift at "4.2.0"
*** Checking out QRCode at "2.0"
*** Checking out Disk at "0.3.3"
*** Checking out Alamofire at "4.7.3"
*** Checking out Toucan at "1.0.0"
*** xcodebuild output can be found in /var/folders/hw/jj1v05q13_52bn_f09w_z_dm0000gn/T/carthage-xcodebuild.FgT5Xh.log
*** Downloading RxSwift.framework binary at "Zürich"
*** Building scheme "Alamofire iOS" in Alamofire.xcworkspace
*** Building scheme "Disk" in Disk.xcworkspace
*** Building scheme "DZNEmptyDataSet" in DZNEmptyDataSet.xcodeproj
*** Building scheme "Lumina" in Lumina.xcworkspace
*** Building scheme "PopupDialog" in PopupDialog.xcworkspace
*** Building scheme "QRCode" in Example.xcodeproj
*** Building scheme "SkeletonView-iOS" in SkeletonView.xcodeproj
*** Building scheme "SVProgressHUD-Framework" in SVProgressHUD.xcodeproj
*** Building scheme "SwiftDate-iOS" in SwiftDate.xcodeproj
Build Failed
        Task failed with exit code 65:
        /usr/bin/xcrun xcodebuild -project /Users/jongbanaag/Documents/Code/App/iStack/chkn-app/Carthage/Checkouts/SwiftDate/SwiftDate.xcodeproj -scheme SwiftDate-iOS -configuration Release -derivedDataPath /Users/jongbanaag/Library/Caches/org.carthage.CarthageKit/DerivedData/9.4.1_9F2000/SwiftDate/5.0.2 -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive -archivePath /var/folders/hw/jj1v05q13_52bn_f09w_z_dm0000gn/T/SwiftDate SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO STRIP_INSTALLED_PRODUCT=NO (launched in /Users/jongbanaag/Documents/Code/App/iStack/chkn-app/Carthage/Checkouts/SwiftDate)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/hw/jj1v05q13_52bn_f09w_z_dm0000gn/T/carthage-xcodebuild.FgT5Xh.log
```

**Comment by drekka** [comment]
Came across this. Appears that there is an issue with Swiftlint failing the build.

**Comment by malcommac** [comment]
Fixed in 5.0.3
